components/Carousel/slick.js

```diff
@@ -0,0 +1,265 @@
+import styled from 'styled-components';
+
+const SlickSliderWrapper = styled.section`
+  padding: 8px;
+
+  .slick-loading .slick-list {
+    background: #fff url('./ajax-loader.gif') center center no-repeat;
+  }
+
+  /* Arrows */
+  .slick-prev,
+  .slick-next {
+    font-size: 0;
+    line-height: 0;
+    position: absolute;
+    top: 50%;
+    display: block;
+    width: 24px;
+    height: 24px;
+    padding: 0;
+    -webkit-transform: translate(0, -50%);
+    -ms-transform: translate(0, -50%);
+    transform: translate(0, -50%);
+    cursor: pointer;
+    border: none;
+    outline: none;
+    background: transparent;
+  }
+  .slick-prev:hover,
+  .slick-prev:focus,
+  .slick-next:hover,
+  .slick-next:focus {
+    color: transparent;
+    outline: none;
```

Comment:
**question:** is that bad a11y?

Reply:
This css is from the lib of react-slick. We have tested and the acessibility was not impaired.